packages/axelar-wasm-std/src/verification.rs

```diff
@@ -0,0 +1,12 @@
+use cosmwasm_schema::cw_serde;
+
+#[cw_serde]
+#[derive(Copy)]
+pub enum VerificationStatus {
+    SucceededOnChain,
+    FailedOnChain,
+    NotFound,
+    FailedToVerify, // verification process failed, e.g. no consensus reached
```

Comment:
```suggestion
    QuorumNotReached
```
or 
```suggestion
    NoQuorum
```
or some such

Reply:
What I don't like about those names is that they apply only to voting, but since this is part of the API it should apply for all kinds of verifiers